diff --git a/node_modules/@digitalbazaar/credentials-context/lib/index.js b/node_modules/@digitalbazaar/credentials-context/lib/index.js
index 6c99a88..524ab6b 100644
--- a/node_modules/@digitalbazaar/credentials-context/lib/index.js
+++ b/node_modules/@digitalbazaar/credentials-context/lib/index.js
@@ -26,7 +26,7 @@ setExportsFromMetadata({
     id: 'https://www.w3.org/ns/credentials/undefined-terms/v2',
     type: 'ContextMetadata',
     shortName: 'undefined-terms-v2',
-    url: new URL('../contexts/undefined-terms-v2.jsonld', import.meta.url),
+    url: null,
     context: undefinedTermsV2Context
   }
 });
@@ -36,7 +36,7 @@ setExportsFromMetadata({
     id: 'https://www.w3.org/2018/credentials/v1',
     type: 'ContextMetadata',
     shortName: 'v1',
-    url: new URL('../contexts/v1.jsonld', import.meta.url),
+    url: null,
     context: v1Context
   }
 });
@@ -46,7 +46,7 @@ setExportsFromMetadata({
     id: 'https://www.w3.org/ns/credentials/v2',
     type: 'ContextMetadata',
     shortName: 'v2',
-    url: new URL('../contexts/v2.jsonld', import.meta.url),
+    url: null,
     context: v2Context
   }
 });
